packages/core/src/sync-store/postgres/format.ts

```diff
@@ -46,7 +45,7 @@ export function rpcToPostgresBlock(
     gasUsed: BigInt(block.gasUsed),
     hash: block.hash!,
```

Comment:
Hmm, I wonder if we'll ever run into issues with inconsistent casing in other hex strings returned by various RPC providers.